ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/MiningParameters.java

```diff
@@ -277,5 +278,10 @@ default long getPosBlockCreationRepetitionMinDuration() {
     default String getStratumExtranonce() {
       return "080c";
     }
+
+    @Value.Default
+    default long getTxsSelectionMaxTime() {
```

Comment:
Because this timeout applies to the whole selection process, so the cumulative time of processing multiple txs, not to a single tx. The timeout for the single tx will be add in a future PR.

Is the option description clear enough?
_Specifies the maximum time, in milliseconds, that could be spent selecting transactions to be included in the block._

Reply:
what about getBlockTxsSelectionMaxTime

Reply:
Yes I think adding _block_ makes it clearer